ursabot/builders.py

```diff
@@ -285,6 +293,20 @@ class ArrowCppTest(Builder):
     ]
 
 
+class ArrowCppBenchmark(Builder):
+    tags = ['arrow', 'cpp']
```

Comment:
Benchmark perhaps?

Reply:
Already implicit via the docker image (I added it here first, but then it complained about duplicate tags).